client/signup/steps/sell-details/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useTranslate } from 'i18n-calypso';
```

Comment:
This step already exists in other flows – was it necessary to do a complete copy/paste? (I don't know, just asking. 🙃 )

Reply:
At first I thought it was not just the copy/imgs changes, but also that the DB field would be different, but taking a better look at it I realized that store name is the siteTitle.

I ended up refactoring the code, adding a couple of props to the existing component and it seems to be working!